packages/shared/sierra-client/src/HttpSierraClient.ts

```diff
@@ -152,15 +157,17 @@ export default class HttpSierraClient implements SierraClient {
         }
       );
 
-      const updatedVarFields = verified
+      const notesVarFields = verified
         ? updateVerificationNote(currentRecordResponse.data.varFields)
-        : currentRecordResponse.data.varFields;
+        : // The way that varFields patches work in the Sierra API means
+          // that PUTting an empty array will be a no-op
```

Comment:
Is this documented behaviour, or is it something we've observed?

It might be worth recording the distinction, so that if it changes we spot it and don't start deleting notes inadvertently.

Reply:
Yeah, good point